tinychat/examples/tinychat/index.html

```diff
@@ -1,111 +1,168 @@
-<!DOCTYPE html>
+<!doctype html>
 
 <head>
-<title>tinychat</title>
-<meta content="width=device-width, initial-scale=1" name="viewport"/>
-<link href="favicon.svg" rel="icon" type="image/svg+xml"/>
-<script defer="" src="/static/cdn.jsdelivr.net/npm/@alpine-collective/toolkit@1.0.2/dist/cdn.min.js"></script>
-<script defer="" src="/static/cdn.jsdelivr.net/npm/@alpinejs/intersect@3.x.x/dist/cdn.min.js"></script>
-<script defer="" src="/static/cdn.jsdelivr.net/npm/@alpinejs/focus@3.x.x/dist/cdn.min.js"></script>
-<script defer="" src="/static/unpkg.com/@marcreichel/alpine-autosize@1.3.x/dist/alpine-autosize.min.js"></script>
-<script defer="" src="/static/unpkg.com/alpinejs@3.x.x/dist/cdn.min.js"></script>
-<script src="/static/unpkg.com/dompurify@3.1.5/dist/purify.min.js"></script>
-<script src="/static/unpkg.com/marked@13.0.0/marked.min.js"></script>
-<script src="/static/unpkg.com/marked-highlight@2.1.2/lib/index.umd.js"></script>
-<script src="/static/unpkg.com/@highlightjs/cdn-assets@11.9.0/highlight.min.js"></script>
-<script src="/index.js"></script>
-<link href="/static/fonts.googleapis.com" rel="preconnect"/>
-<link crossorigin="" href="/static/fonts.gstatic.com" rel="preconnect"/>
-<link href="/static/fonts.googleapis.com/css2" rel="stylesheet"/>
-<link href="/static/cdn.jsdelivr.net/npm/purecss@3.0.0/build/base-min.css" rel="stylesheet"/>
-<link crossorigin="anonymous" href="/static/cdnjs.cloudflare.com/ajax/libs/font-awesome/6.5.2/css/all.min.css" integrity="sha512-SnH5WK+bZxgPHs44uWIX+LLJAJ9/2PkPKZ5QiAj6Ta86w+fsb2TkcmfRyVX3pBnMFcV7oQPJkl9QevSCWr3W6A==" referrerpolicy="no-referrer" rel="stylesheet">
-<link href="/static/unpkg.com/@highlightjs/cdn-assets@11.9.0/styles/vs2015.min.css" rel="stylesheet"/>
-<link href="/index.css" rel="stylesheet"/>
-<link href="/common.css" rel="stylesheet"/>
-</link></head>
```

Comment:
Does prettier not want the closing link and head tag on separate lines?

Reply:
The `link` tag is self closing and does not require a closing tag. Hence, there is no need for the `<\link> tag.

Even if you put them on separate lines, the issue of not formatting the code still persists.

So only thing we can do it to get rid of it.